src/Sylius/Bundle/ResourceBundle/spec/Form/EventSubscriber/AddCodeFormSubscriberSpec.php

```diff
@@ -76,4 +77,34 @@ function it_throws_exception_when_resource_does_not_implement_code_aware_interfa
         $event->getData()->willReturn($object);
         $this->shouldThrow('\UnexpectedTypeException');
     }
+
+    function it_sets_type_as_currency_when_resource_implements_currency_interface(FormEvent $event, FormInterface $form, CurrencyInterface $resource)
```

Comment:
Please remove these 2 examples and update the whole spec to take into account different form types.